atomic

```diff
@@ -395,12 +395,16 @@ def create_parser(atomic):
             ('yes', 'true', 't', 'y', '1') else False
     scanners = get_scanners()
     atomic_config = get_atomic_config()
-    if atomic_config is None or atomic_config.get('default_scanner') is None:
-        sys.stderr.write("\nYou must have a default scanner defined in /etc/atomic.conf. Use --list to see "
-                         "what scanners are available for your system.\n")
+    if atomic_config is None or (atomic_config.get('default_scanner') is None and len(scanners) > 1):
+        sys.stderr.write("\nBecause you have multiple scanners configured for your system, you must have a default "
+                         "scanner defined in /etc/atomic.conf. Use --list to see "
```

Comment:
You have multiple scanners configured on your system, but you did not define a default in /etc/atomic.conf.  You need to specify the scanner you wish to execute.  Use --list to show all installed scanners.